frontend/src/metabase/visualizations/visualizations/SmartScalar.jsx

```diff
@@ -21,17 +21,67 @@ import {
   getDefaultSize,
   getMinSize,
 } from "metabase/visualizations/shared/utils/sizes";
+
+import { measureText } from "metabase/lib/measure-text";
+import { space } from "metabase/styled-components/theme";
 import { isDate } from "metabase-lib/types/utils/isa";
 import { ScalarContainer } from "./Scalar.styled";
 
 import {
+  PreviousValue,
   PreviousValueContainer,
   PreviousValueSeparator,
-  PreviousValueVariation,
+  PreviousValueWrapper,
+  Separator,
   Variation,
+  VariationIcon,
+  VariationTooltip,
+  VariationValue,
 } from "./SmartScalar.styled";
 
-export default class Smart extends Component {
+const SPACING = parseInt(space(1), 10);
```

Comment:
I believe we shouldn't rely on `space` returning pixels. I'd rather put an explicit `8` constant here instead

Reply:
aea584d13d4292b7a81bfc86cce3e4567603fee2